templates/blocks/share_buttons.twig

```diff
@@ -1,41 +1,38 @@
 {% block javascripts %}
-    <script>
-        window.addEventListener('DOMContentLoaded', () => {
```

Comment:
I don't necessary to wait the DOM content loaded event, since we'll need to have all the functionality ready at the same time the buttons are added into the DOM. If you don't do that, you'll take the risk to get `undefined` to `window.dataLayerPush` function.